ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -3903,6 +3903,7 @@ signal_list_t *create_gate(ast_node_t* gate, char *instance_name_prefix, sc_hier
 		    /* hookup the output pins */
 		    hookup_output_pins_from_signal_list(gate_node, 0, out_1, 0, 1);
 
+			free_signal_list(out_1); 
```

Comment:
Move the free to the start of the for loop and set out_1 to NULL afterwards so that the check done lower (oassert) can work.

Reply:
Fix your indentation